.ruby-version

```diff
@@ -1 +1 @@
-2.7.2
\ No newline at end of file
+2.7.7
```

Comment:
Is this necessary as part of this PR?

Reply:
This is the only 2.7 version I can get to work on my machine -- I'll revert this